sgx-tools/proto/aesm-service.proto

```diff
@@ -0,0 +1,90 @@
+syntax = "proto3";
+
+package dcap_aesm_service; // import "github.com/inclavare-containers/sgx-tools/proto"
+
+message AesmServiceRequest{
```

Comment:
There is a space before '{' according to the style of https://github.com/alibaba/inclavare-containers/blob/master/rune/libenclave/intelsgx/proto/aesm-service.proto.